Mega - failure if public-private key creation" was not done.

Trying to list mega account size (using rclone browser and rclone1.42 branch), while the "Public - Private key creation" has not been done in mega, produces :

panic: runtime error: index out of range

goroutine 1 [running]:
github.com/ncw/rclone/vendor/github.com/t3rm1n4l/go-mega.getMPI(0x1816830, 0x0, 0x0, 0x1816830, 0x0, 0x0, 0x0)
	C:/gopath/src/github.com/ncw/rclone/vendor/github.com/t3rm1n4l/go-mega/utils.go:174 +0x126
github.com/ncw/rclone/vendor/github.com/t3rm1n4l/go-mega.decryptSessionId(0xc042299898, 0x0, 0x20, 0xc042299878, 0x0, 0x20, 0xc0423bf0e0, 0x10, 0x12, 0x1506a8, ...)
	C:/gopath/src/github.com/ncw/rclone/vendor/github.com/t3rm1n4l/go-mega/utils.go:244 +
0x154
github.com/ncw/rclone/vendor/github.com/t3rm1n4l/go-mega.(*Mega).Login(0xc04237e0e0, 0xc0420a29c7, 0x31, 0xc0423328e0, 0xf, 0x0, 0xc0420a6050)
	C:/gopath/src/github.com/ncw/rclone/vendor/github.com/t3rm1n4l/go-mega/mega.go:493 +0x4da
github.com/ncw/rclone/backend/mega.NewFs(0xc0420a6050, 0x47, 0xc0420a6050, 0x0, 0x11c3820, 0xc0423700c0, 0x0, 0x0, 0x0, 0x0)
	C:/gopath/src/github.com/ncw/rclone/backend/mega/mega.go:200 +0x5fb
github.com/ncw/rclone/fs.NewFs(0xc0420a6050, 0x48, 0xc04205d200, 0xc0420a6050, 0x47, 0xc0420a6050)
	C:/gopath/src/github.com/ncw/rclone/fs/fs.go:1002 +0xbc
github.com/ncw/rclone/cmd.NewFsFile(0xc0420a6050, 0x48, 0xc042360800, 0xc042368000, 0xc042299be0, 0x5e1bf7)
	C:/gopath/src/github.com/ncw/rclone/cmd/cmd.go:161 +0x99
github.com/ncw/rclone/cmd.newFsFileAddFilter(0xc0420a6050, 0x48, 0xc042299c98, 0xc0420ca800, 0xc042087d10, 0xc042299cc0)
	C:/gopath/src/github.com/ncw/rclone/cmd/cmd.go:179 +0x54
github.com/ncw/rclone/cmd.NewFsSrc(0xc042087dd0, 0x1, 0x3, 0xc042087dd0, 0x1)
	C:/gopath/src/github.com/ncw/rclone/cmd/cmd.go:201 +0x48
github.com/ncw/rclone/cmd/size.glob..func1(0x17e7f40, 0xc042087dd0, 0x1, 0x3)
	C:/gopath/src/github.com/ncw/rclone/cmd/size/size.go:26 +0x88
github.com/ncw/rclone/vendor/github.com/spf13/cobra.(*Command).execute(0x17e7f40, 0xc042087d10, 0x3, 0x3, 0x17e7f40, 0xc042087d10)
	C:/gopath/src/github.com/ncw/rclone/vendor/github.com/spf13/cobra/command.go:766 +0x2c8
github.com/ncw/rclone/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x17dfa40, 0x0, 0x0, 0x0)
	C:/gopath/src/github.com/ncw/rclone/vendor/github.com/spf13/cobra/command.go:852 +0x311
github.com/ncw/rclone/vendor/github.com/spf13/cobra.(*Command).Execute(0x17dfa40, 0x0, 0x17e88c0)
	C:/gopath/src/github.com/ncw/rclone/vendor/github.com/spf13/cobra/command.go:800 +0x32
github.com/ncw/rclone/cmd.Main()
	C:/gopath/src/github.com/ncw/rclone/cmd/cmd.go:545 +0x3d
main.main()
	C:/gopath/src/github.com/ncw/rclone/rclone.go:14 +0x27

It is normal that it is unable to get the information or to work properly, but perhaps failure can be avoided and information provided about the error...

Thanks,
CT

--- a/docs/content/mega.md
+++ b/docs/content/mega.md
@@ -68,6 +68,9 @@ d) Delete this remote
 y/e/d> y
 ```
 
+**NOTE:** The encryption keys need to have been already generated after a regular login
+via the browser, otherwise attempting to use the credentials in `rclone` will fail.
+
 Once configured you can then use `rclone` like this,
 
 List directories in top level of your Mega
@@ -152,11 +155,9 @@ permanently delete objects instead.
 
 ### Limitations ###
 
-This backend uses the [go-mega go
-library](https://github.com/t3rm1n4l/go-mega) which is an opensource
+This backend uses the [go-mega go library](https://github.com/t3rm1n4l/go-mega) which is an opensource
 go library implementing the Mega API. There doesn't appear to be any
-documentation for the mega protocol beyond the [mega C++
-SDK](https://github.com/meganz/sdk) source code so there are likely
-quite a few errors still remaining in this library.
+documentation for the mega protocol beyond the [mega C++ SDK](https://github.com/meganz/sdk) source code
+so there are likely quite a few errors still remaining in this library.
 
 Mega allows duplicate files which may confuse rclone.

